source/numerics/solution_transfer.inst.in

```diff
@@ -22,5 +22,9 @@ for (VEC : VECTOR_TYPES; deal_II_dimension : DIMENSIONS;
       deal_II_dimension,
       VEC,
       DoFHandler<deal_II_dimension, deal_II_space_dimension>>;
+    template class SolutionTransfer<
+      deal_II_dimension,
+      VEC,
+      hp::DoFHandler<deal_II_dimension, deal_II_space_dimension>>;
```

Comment:
I'm curious - why was this necessary to revert?

Reply:
The problem is that `SolutionTransfer` is templated with `DoFHandlerType` and the test have complained about the undefined references. For the same reason, a lot functions has to be instantiated for both dof-handler types...

Removing the `DoFHandlerType` template argument from the library as listed in https://github.com/dealii/dealii/issues/10333#issue-623676096 would enable us to not instantiate tor the dummy `hp::DoFHandler` (however this might be backwards incompatibility!?).

Reply:
Oh, I see. Lets address that later.